pkg/lightning/backend/tidb/tidb.go

```diff
@@ -287,6 +296,22 @@ func (*targetInfoGetter) CheckRequirements(ctx context.Context, _ *backend.Check
 	return nil
 }
 
+// stmtKey defines key for stmtCache.
+type stmtKey struct {
+	query string
+	// `hash` is the hash value of this object.
+	hash []byte
+}
+
+// Hash implements SimpleLRUCache.Key.
+func (k *stmtKey) Hash() []byte {
```

Comment:
(I'm not sure why the `kvcache.Key` interface call it `Hash`. Hash means a fixed-length digest)